src/entities/purchase-params.ts

```diff
@@ -21,4 +22,21 @@ export interface PurchaseParams {
    * The email of the user. If undefined, RevenueCat will ask the customer for their email.
    */
   customerEmail?: string;
+
+  /**
+   * Optional customization for the translations used in the purchase flow.
+   * The keys are the locale codes and the values are objects with the same keys as the default translations.
+   */
+  customTranslations?: CustomTranslations;
```

Comment:
I was chatting about this with @aboedo today. I haven't been on the conversations that lead to this new API but one thought that came was that, instead of having a client-side API to override the strings, it might be a good idea to provide all texts from the backend, so they can be dynamically configured from there if needed. Specially with the localization changes we've been working on for paywalls, this might be much easier. 

I thought this idea could be great, but not sure if we would have the bandwidth to do it right now... Having said that, a new public API would be trickier to remove once it's published, so if we want to go that route, it might be worth considering not adding this parameter yet (not sure how much of a priority it has)

Reply:
I think there are pro and cons to forcing backend customization only.

Long term we want to allow developers to customize their labels, but on the mid-short term we won't be able to build an editor right now. I think this is a compromise that would unlock customers to customize the current version of the SDK now.

I see the backend and hardcoded version living together as follows
````
const finalLabels= {
   ...defaultTranslations,
   ...serverSideCustomTranslations,
   ...customTranslationsPassedAsParameter
}
```

In this way, if someone locally wants to generate something dynamically in their app with their logic for one label they can. I also think that who uses the sdk directly should be able to update their website relatively easily and fix the `customTranslations` object.

I see how the problem might be bigger on mobile where you don't have control on which version of the app is being executed on the phone even if you update it, but on web this is a lesser problem IMO.

I am fine removing this public parameter, however, this is a direct customer request that we had from Dipsea.
We do not need it for the Paywalls ship so we can definitely try to discuss it more and exclude it from this PR.

Reply:
Removed the public `customTranslations?: CustomTranslations;` parameter for now.

We can iterate again when we'll work on customizing the WPL (which was the original request from Dipsea).